gateway/1.4/modules/ROOT/pages/flex-architecture-basic-deployments.adoc

```diff
@@ -0,0 +1,41 @@
+= Basic Deployment Models for Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway supports three basic deployment models, which apply to both Connected Mode and Local Mode:
+
+* <<standalone-deployment, Standalone Deployment>>
+* <<ingress-deployment, Ingress Deployment>>
+* <<sidecar-deployment, Sidecar Deployment>>
+
+The deployment models in this overview use Kubernetes terminology, but you can extend the deployment models to other technological stacks that have the appropriate network controls applied.
+
+== Standalone Deployment
+
+In a standalone deployment, Flex Gateway acts as a standalone service that protects one or more integration API flows by managing internal traffic. In this type of deployment, you deploy Flex Gateway in a separate pod and namespace from the consumer applications:
+
+image:ad-standalone-deployment.png[In a standalone deployment, Flex Gateway and the consumer applications are in different Kubernetes namespaces.]
```

Comment:
I forgot to add this global edit - all alt text that has spaces or punctuation must be enclosed in quotes. Gary says that it doesn't render correctly otherwise.